scripts/batch-score.coffee

```diff
@@ -0,0 +1,107 @@
+# Description:
+#   Script for batch wise score.
+#
+# Configuration:
+#   INFO_SPREADSHEET_URL
+#
+# Commands:
+#   hubot score bxx
+# 
+# Author:
+#   aseem09
+
+module.exports = (robot) ->
+
+  # using this, stores the data in brain's "scorefield" key
+  scorefield = () ->
+    Field = robot.brain.get("scorefield") or {}
+    robot.brain.set("scorefield",Field)
+    Field
+  
+  # response for score status of any <batch>
+  stringLength = (str) ->
+    string=String(str)
+    temp=string.split('')
+    return temp.length
+  
+  pad = (array) ->
```

Comment:
May add a comment that tells what `pad` is doing here, i.e., it makes every element the same "width".